spec/duration.html

```diff
@@ -419,7 +419,12 @@ <h1>Temporal.Duration.prototype.round ( _options_ )</h1>
         1. Perform ? RequireInternalSlot(_duration_, [[InitializedTemporalDuration]]).
         1. If _options_ is *undefined*, then
           1. Throw a *TypeError* exception.
-        1. Set _options_ to ? GetOptionsObject(_options_).
+        1. If Type(_options_) is String, then
+          1. Let _optionString_ be _options_.
+          1. Set _options_ to ! OrdinaryObjectCreate(*null*).
+          1. Perform ! CreateDataPropertyOrThrow(_options_, *"_smallestUnit_"*, _optionString__).
```

Comment:
Yep, that's the current behavior of `Duration.p.round`. (It doesn't apply to any other methods). This PR is narrowly focused on the string parameter and doesn't change any existing options behavior.

The champions' consensus from this morning was that it was a good idea to let developers use a string in cases where an options bag is currently required. This change makes the API more ergonomic for the most common use cases without making uncommon use cases harder. And it addresses what I understood to be the highest-priority concern from #1756, which is that an options bag should actually be optional.  Thanks for pushing us to find this win-win solution!

Changing existing behavior of options is a more complicated discussion, with some tradeoffs involved. We're open to having that discussion, but for now we wanted to lock in benefits that everyone agrees with in time for the Oct plenary.

Tomorrow I'll open a new issue for your suggestion above to make `smallestUnit` required in `duration.p.round`, and we can discuss there. Sound OK?

Reply:
I think that in the case where there is a *single* required argument, this overall pattern is a win.

I think that in the case where there's something more complex - like this case - allowing the string is actually more confusing, because "which required thing is it, smallest or largest?" I'm not on board with this change for `round` (assuming round is the only method in this PR where it's not "precisely one property in the options bag is required").

Reply:
> Tomorrow I'll open a new issue for your suggestion above to make `smallestUnit` required in `duration.p.round`, and we can discuss there.

https://github.com/tc39/proposal-temporal/issues/1876